chore: deprecate `NgZorroAntdModule` (#4519)
NG-ZORRO/ng-zorro-antd: README.md

## 🔨 Usage

> Import the component modules you want to use into your `app.module.ts` file or other lazy-loaded feature modules.

Reviewer: ```suggestion
Import the component modules you want to use into your `app.module.ts` file and other feature modules.
```
